modules/conductor/pom.xml

```diff
@@ -89,6 +89,10 @@
       <groupId>commons-io</groupId>
       <artifactId>commons-io</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Not sure why this is suddenly required, but Maven says it's an undeclared usage if we don't have it.